src/content_api/content_api_extract.py

```diff
@@ -123,11 +59,12 @@ def chunked_extract(nodes_srs, chunks_list, destination_dir):
                 json_dict[node] = content_item
```

Comment:
I can't see if `content_item['url']` is used anywhere

Reply:
ah unless this is catching multiple paths for a content ID?

Reply:
Yep